tutorials/index.md

```diff
@@ -27,3 +29,18 @@ Python:
 -   [Case Study: Analyzing Genomic Data with Biomedical Data Commons](https://colab.research.google.com/github/datacommonsorg/api-python/blob/master/notebooks/analyzing_genomic_data.ipynb){:target="_blank"}
 
 -   [Case Study: Drug Discovery with Biomedical Data Commons](https://colab.research.google.com/github/datacommonsorg/api-python/blob/master/notebooks/Drug_Discovery_With_Data_Commons.ipynb){:target="_blank"}
+
+
+### India
+-   [Analysis of Mean Daily Wages of Men and Women of India](https://colab.research.google.com/drive/1oL0v6ZB4InHnMYo3bTQcSs4ZIHz6XIQ7#scrollTo=GE0b7K-Tiq8p){:target="_blank"}
```

Comment:
See my comments on a copy of this colab here:https://colab.research.google.com/drive/1za7WMutuAJT4JdhVmn2CviQ4yJVGm-1p?usp=sharing

Most importantly, there's a section that's broken because you reference a local .csv file